drivers/hmis/app/controllers/hmis/graphql_controller.rb

```diff
@@ -13,14 +13,31 @@ class GraphqlController < Hmis::BaseController
 
     def execute
       variables = prepare_variables(params[:variables])
-      query = params[:query]
-      operation_name = params[:operationName]
       context = {
         current_user: current_hmis_user,
       }
       case params[:schema]
       when :hmis
-        result = HmisSchema.execute(query, variables: variables, context: context, operation_name: operation_name)
+        if params[:_json]
+          # We have a batch of operations
```

Comment:
Is there a way to test making batch requests from graphiql?

Reply:
Not that I'm aware of. I did test it with the frontend though by loading multiple queries in a single component. We could also write a test for it.